src/components/SelectionList/BaseListItem.tsx

```diff
@@ -40,6 +40,7 @@ function BaseListItem<TItem extends ListItem>({
     hoverStyle,
     onLongPressRow,
     testID,
+    shouldUseDefaultRightHandSideCheckmark = true,
```

Comment:
I wonder though if there is a better long term solution than this one? @daledah @thelullabyy